src/libYARP_dev/src/devices/AnalogWrapper/AnalogWrapper.h

```diff
@@ -94,17 +100,28 @@ namespace yarp{
  * frame_id      r_shoulder
  * \endcode
  *
- * Configuration file using .xml format.
+ * Configuration file using .xml format(skin example).
```

Comment:
I would not use a skin example in the documentation of the analogServer.
Please remove all the references to skin in this example and just use it as a plain multichannel analog sensor.